extensions/SemanticMediaWiki/includes/specials/SMW_SpecialSMWAdmin.php

```diff
@@ -74,17 +74,18 @@ public function execute( $par ) {
 
 		$this->messageBuilder = $this->htmlFormRenderer->getMessageBuilder();
 
-		#$jobQueueLookup = $mwCollaboratorFactory->newJobQueueLookup( $this->getStore()->getConnection( 'mw.db' ) );
-		#$row = $jobQueueLookup->selectJobRowFor( 'SMW\RefreshJob' );
-		#
-		#if ( $row !== false ) { // similar to Job::pop_type, but without deleting the job
-		#	$title = Title::makeTitleSafe( $row->job_namespace, $row->job_title );
-		#	$blob = (string)$row->job_params !== '' ? unserialize( $row->job_params ) : false;
-		#	$refreshjob = Job::factory( $row->job_cmd, $title, $blob, $row->job_id );
-		#} else {
-		#	$refreshjob = null;
-		#}
-		$refreshjob = null; // Wikia change - we do not use job DB table for handling tasks queue (SUS-1305)
+		// $jobQueueLookup = $mwCollaboratorFactory->newJobQueueLookup( $this->getStore()
+		//	->getConnection( 'mw.db' ) );
+		// $row = $jobQueueLookup->selectJobRowFor( 'SMW\RefreshJob' );
+
+		// if ( $row !== false ) { // similar to Job::pop_type, but without deleting the job
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this commented out code. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS125)